ci-operator/step-registry/cucushift/upgrade/disconnected/cucushift-upgrade-disconnected-workflow.yaml

```diff
@@ -0,0 +1,14 @@
+workflow:
+  as: cucushift-upgrade-disconnected
+  steps:
+    test:
+      - ref: cucushift-pre
+      - ref: cucushift-upgrade-pre     
+      - ref: cucushift-upgrade-mirror-images-by-oc-adm
```

Comment:
Why not reuse this step `mirror-images-by-oc-adm`? Here: https://github.com/openshift/release/tree/master/ci-operator/step-registry/mirror-images/by-oc-adm

Reply:
This was discussed in https://github.com/openshift/release/pull/28931#discussion_r884389773